README.md

```diff
@@ -9,6 +9,7 @@
 [![Twitter][twitter-badge]][twitter-link]
 [![Matrix][matrix-badge]][matrix-link]
 
+<p align="center"><img src="https://raw.githubusercontent.com/RIOT-OS/RIOT/master/doc/doxygen/src/riot-logo.svg" width="66%"><!--
```

Comment:
While squashing, can you at least try

```suggestion
<p align="center"><img src="doc/doxygen/src/riot-logo.svg" width="66%"><!--
```

first? AFAIK this should work (yes I know your tool says otherwise ;-)).

Reply:
This works indeed!